server/common/app_config.py

```diff
@@ -21,6 +21,18 @@
 # anything bigger than this will generate a special message
 BIG_FILE_SIZE_THRESHOLD = 100 * 2 ** 20  # 100MB
 
+""" Default quotas """
+Default_Quotas = {
```

Comment:
I think the client needs to be aware of these limits so that we can provide proper feedback to the user.  Right now I think they just get back "BadRequest" if a limit is exceeded.   I suggest that we add these quotas to the config endpoint.

Reply:
I see the response does have some descriptive information about the error.  But I still suggest that we need an tracker issue to tell the client what the limits are so the client code to warn the user that they've selected too many cells, for example.

Reply:
True. Right now, I picked defaults that reflect what the front-end is actually doing, on the assumption we could co-evolve these.  If they ever become externally configurable, letting the client know is critical.

I'll go ahead and add to the config route for now.  Simple change...